src/components/ListDetail/ListDetail.module.css

```diff
@@ -167,49 +174,58 @@
 
 /**
 * List detail number
-* 1) Used on list items using number variant
+* 1) Used on list items using number variant; provides a cosmetic cicular border
+* 2) Using flex to keep the number centered
+* 3) Since this will display as a cirlce, keep height and width equal; should be wide enough to fit a two-digit number
+* 4) Using a percentage here instead of --eds-theme-border-radius because this is not a typical bordered object
 */
 .list-detail__number {
   position: relative;
+  display: flex; /* 2 */
+  justify-content: center;
+  align-items: center;
   left: -4px;
-  width: 12px;
-  height: 12px;
-  border-radius: 50%;
-  border: 1px solid var(--eds-theme-color-neutral-subtle-border);
+  width: 1.5rem; /* 3 */ 
+  height: 1.5rem; /* 3 */
+  border-radius: 50%; /* 4 */
+  border: 1px solid var(--eds-theme-color-border-neutral-subtle);
   margin-right: 9px;
-  padding: 4px 8px;
-  font-size: 12px;
+  font-size: var(--eds-font-size-12);
   margin-left: -4px;
-  background: var(--eds-color-neutral-white); /* 1*/
+  background: var(--eds-theme-color-background-neutral-default); 
 }
 
 /**
 * List detail icon
-* 1) Used so the transparent icon doesn't show the line behind. TODO: Map to tier 2
+* 1) Used so the transparent icon doesn't show the line behind. 
 */
 .list-detail__icon {
   position: relative;
   left: -7px;
   height: var(--eds-size-3);
   width: var(--eds-size-3);
   margin-right: var(--eds-size-half);
-  background: var(--eds-color-neutral-white); /* 1*/
+  background: var(---eds-theme-color-background-neutral-default); /* 1*/
   z-index: var(--eds-z-index-100);
 
   /**
   * List detail icon within success list detail item
   */
   .list-detail__item--success & {
-    fill: var(--eds-theme-color-utility-success-foreground);
-  } /**
+    fill: var(--eds-theme-color-background-grade-complete-default);
+  } 
+  
+  /**
   * List detail icon within warning list detail item
   */
   .list-detail__item--warning & {
-    fill: var(--eds-theme-color-utility-warning-foreground);
-  } /**
+    fill: var(--eds-theme-color-background-grade-revise-default);
```

Comment:
The exclamation mark used in this icon should be black. Adding that flexibility to the Icon component seems like the best way to handle it? Adding a workaround here like a black background color doesn't work because the bounding box is rectangular; getting around that by setting border-radius still results in the background being larger than the svg and giving the appearance of a black border. I'm open to suggestions here, or do we just mark it as tech debt?

Reply:
is "text color" in this case white? I wonder if https://github.com/chanzuckerberg/edu-design-system/pull/940 will make this easier to handle

Reply:
I'm about to review #940 , that would be great if it did